src/index.js

```diff
@@ -56,7 +56,18 @@ module.exports = {
     'sort-vars': ['error'], // https://eslint.org/docs/rules/sort-vars
 
     // Unicorn
+    'unicorn/consistent-destructuring': 'off',
```

Comment:
Why? (just curious)

Reply:
Consistency was lacking in the codebases I worked in.

It's something I'm willing to enforce, but only if everyone's on board. :-)

Reply:
I would leave it on!

Reply:
I would also leave it on.